media/js/helpfulness.js

```diff
@@ -0,0 +1,84 @@
+(function(win, doc, $) {
+    // no localStorage, no helpfulness rating
+    if (('localStorage' in win)) {
+        var stfu = localStorage.getItem('helpful-stfu') === 'true'; // true if ever clicked stfu
```

Comment:
For the sake of positivity, we should probably change the `stfu` references to `ignore` or something like that.  `stfu` may reflect bad on us.